doc/decisions/STEPS.md

```diff
@@ -0,0 +1,144 @@
+   5. If there is any change in the direction of the decision then the PR cannot be merged as "Decided". The decision must at least be moved back to "In Review".
+3. After a decision is merged as "Decided", it can be moved to "Partially Implemented" or "Implemented" by any PR.
+
+The rest of the document describes the steps in full details.
+
+## Drafts
+
+> This step is highly recommended and it is even required if the problem is not yet clear to all the core developers.
+
+The first step is to create a PR with:
+
+- **one** decision, where at least the "Problem" is filled out and "Decision", "Rationale" and "Implications" are **not** yet filled out.
+- optional backlinks from related decisions.
+
+This step is brainstorming for completely new ideas.
+
+> No agreement about anything is needed to merge decisions in this stage.
+
+## Problem Clear
+
+This step is very important:
+
+- it clarifies the importance of the problem, answering:
+  Why should we put our time and energy in this problem and not in another problem.
+- it clarifies the scope of the decision.
+- it clarifies relation to other problems.
+
+Decisions will have much smoother further steps if this step is done carefully without prejudice.
+It is especially important that one shouldn't have a fixed mind-set about a preferred solution from the beginning.
+
+> Everyone must agree that the problem exists and is worth solving so that a decision PR in "Problem Clear" step can be merged.
+> A problem is clear if everyone would be able to describe an experiment or test case that shows if a solution fixes the problem.
+
+## Solutions Clear
+
+> This step is recommended if it is not yet clear to the core developers which solution is the best.
+
+Here you must ensure:
+
+- problem, constraint and assumptions are well-explained and sound
+- links from/to related decisions are created
+- there are several solutions described, each with rationale and implication
+- "Decision", "Rationale" and "Implications" are **not** yet filled out if there are people arguing for different options
+
+Here the decision should not only have one decision but should describe several solutions.
+For each solution a proposal, rationale and optionally implications should be given.
+
+> The solutions are clear if all reviewers understand the given solutions and no reviewer can come up with better solutions.
+> The solution space is clear.
+> I.e. the trade-offs, combinations and pros/cons of the solutions are explored.
+> Decision author and reviewers are satisfied that every useful solution is present in the decision.
+
+## In Review
+
+Now it is allowed to have the decision from the previous round in the "Decision" section.
+In this step, the last details of the chosen decision get polished:
+
+- consistency with other decisions
+- links from/to related decisions checked
+- "Decision", "Rationale" and "Implications" are fully filled out
```

Comment:
IMO this is just describes the state of an **open/unmerged** PR that wants to move/add a decision file to `doc/decisions/X_decided`. I don't really see a reason to ever merge a file in this state, _without_ also having the reviews and committing to the decision.

To me merging a fully filled out decision file in `doc/decisions/Y_in_review` seems like a way of creating a bias towards the decision the author wants. If the reviews already agree, then there is no need for this step. If reviews disagree merging the "Decision" section anyway seems kinda wrong.

Reply:
> merge a file in this state, without also having the reviews and committing to the decision.

There might be plenty of reasons, like one of the reviewers is not available anymore.

> seems like a way of creating a bias towards the decision the author wants

Yes but in this case it is allowed because there were already (hopefully unbiased) discussions showing this clear tendency.